crates/jstzd/tests/octez_client_test.rs

```diff
@@ -69,5 +72,40 @@ async fn generates_keys_throws() {
     let alias = "test_alias".to_string();
     let _ = octez_client.gen_keys(&alias, None).await;
     let res = octez_client.gen_keys(&alias, None).await;
-    assert!(res.is_err_and(|e| e.to_string().contains("failed to generate key")));
+    assert!(res.is_err_and(|e| { e.to_string().contains("\"gen\" \"keys\"") }));
+}
+
```

Comment:
Maybe add 1 more test where the gen keys is called with the signature field set. Probably should have been covered in the gen keys PR

Reply:
done